server/lib/email.js

```diff
@@ -243,6 +243,14 @@ const generateEmailFromTemplate = (template, recipient, data = {}, options = {})
   if (template === 'ticket.confirmed') {
     // if (slug === 'sustainoss') template += '.sustainoss';
     if (slug === 'fearlesscitiesbrussels') template += '.fearlesscitiesbrussels';
+
+    if (slug === 'drupalatx') {
+      const eventSlug = get(options, 'event.slug');
```

Comment:
I haven't tested but seems to me that `event` will never be found in`options` but in `data`.


See:
https://github.com/opencollective/opencollective-api/blob/master/server/lib/notifications.js#L186
https://github.com/opencollective/opencollective-api/blob/master/server/lib/notifications.js#L147

Reply:
Thanks, I thought I tested locally, but must've used the wrong method. It seems like `data` is the right one. I updated the branch.